scripts/ci-install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
In chatting about this, we're pretty sure it's best to converge around the acceptance tests here as they'll cover multiple versions of the provider, and we can matrix test various versions of Terraform itself if needed.